src/main/java/uk/gov/digital/ho/pttg/hmrc/HmrcClient.java

```diff
@@ -70,7 +73,7 @@ public AccessCodeHmrc getAccessCodeFromHmrc(String totpCode) {
             }
         }
 
-        log.info("Received access code response");
+        log.info("Received access code response", value(EVENT, HMRC_ACCESS_CODE_RESPONSE_SUCCESS));
```

Comment:
We've now got the HMRC_ACCESS_CODE_RESPONSE_SUCCESS used in two places in the code, meaning it'll look like we've got double when we search for this Event ID. We use this event to signify that the call to this service (hmrc-access-code) was successful. We need a slightly different event to signal that the call from this service to HMRC was successful.

Reply:
Resolved this by adding a new log event: HMRC_GET_ACCESS_CODE_SUCCESS